docs/Features/XMLFeatures.md

```diff
@@ -0,0 +1,144 @@
+# XML Features
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides a number of notable features, with demos below.
+<!--  -->
+
+## XML Tag Auto Close
+
+XML tag `auto-close` is supported:
+
+Using `/` in an opening tag will auto close the tag.
+
+![Auto Close Tag Slash](../images/Features/AutoCloseTagSlashXML.gif)
+
+<!-- Using a `>` at the end of an opening tag generates the corresponding closing tag.
+
+![Auto Close Tag End](../images/Features/AutoCloseTagEndXML.gif) -->
+
+<!--  -->
+
+## Linked Editing
+
+Linked editing is supported, allowing for simultaneous changes an opening and closing tag pair.
+
+![Linked Editing](../images/Features/LinkedEditingXML.gif)
+
+<!--  -->
+
+## Jump to Start/End Tag
+
+For quick navigation, you can use `Ctrl + Click` on a closing tag to jump to the corresponding opening tag.
+
+![Jump To Opening Tag](../images/Features/JumpToOpeningTagXML.gif)
+
+The same applies to jumping from an opening tag to the respective closing tag.
+
+![Jump To Closing Tag](../images/Features/JumpToClosingTagXML.gif)
+
+<!--  -->
+
+## Rename Tag
+
+To rename a single tag and its corresponding opening/closing, highlight the tag and press `F2` or use the dropdown to select `Rename Symbol`.
+
+![Rename Tag](../images/Features/RenameTagXML.gif)
+
+<!--  -->
+
+## Completion Support
+
+### Basic completion
+
+If an XML file is not associated to a [bound](https://en.wikipedia.org/wiki/XML_data_binding) grammar file, you can utilize XML completion which uses existing XML tag elements.
```

Comment:
No need to link to data binding, I was just trying to explain why connecting an XML document with a schema is sometimes called "binding" to a grammar file.